src/components/Card/Card.jsx

```diff
@@ -7,15 +7,21 @@ import {
   CardImage,
   CardContent,
 } from './Card.styled';
-import { useVideo } from '../../providers/Video/Video.provider';
+import { useVideo } from '../../providers/Video/provider';
 
-const Card = ({ video }) => {
+const Card = ({ video, isFavorites }) => {
```

Comment:
I changed this prop's name in the last push, I renamed to `isFavoritesPage` because it's a flag that I am using to identify the parent component and define the route for the video when is clicked